src/libs/SearchUtils.ts

```diff
@@ -32,25 +50,35 @@ function getTransactionsSections(data: OnyxTypes.SearchResults['data']): Transac
     const shouldShowCategory = getShouldShowColumn(data, CONST.SEARCH_TABLE_COLUMNS.CATEGORY);
     const shouldShowTag = getShouldShowColumn(data, CONST.SEARCH_TABLE_COLUMNS.TAG);
     const shouldShowTax = getShouldShowColumn(data, CONST.SEARCH_TABLE_COLUMNS.TAX_AMOUNT);
+
     return Object.entries(data)
         .filter(([key]) => key.startsWith(ONYXKEYS.COLLECTION.TRANSACTION))
-        .map(([, value]) => {
-            const isExpenseReport = value.reportType === CONST.REPORT.TYPE.EXPENSE;
+        .map(([, transactionItem]) => {
+            const isExpenseReport = transactionItem.reportType === CONST.REPORT.TYPE.EXPENSE;
+            const from = data.personalDetailsList?.[transactionItem.accountID];
+            const to = isExpenseReport
+                ? (data[`${ONYXKEYS.COLLECTION.POLICY}${transactionItem.policyID}`] as SearchAccountDetails)
+                : (data.personalDetailsList?.[transactionItem.managerID] as SearchAccountDetails);
+
+            const formattedFrom = from.displayName ?? from.login ?? '';
+            const formattedTo = to.name ?? to.displayName ?? to.login ?? '';
```

Comment:
```suggestion
            const formattedTo = to?.name ?? to?.displayName ?? to?.login ?? '';
```

https://github.com/Expensify/App/assets/102477862/e5e48a9c-90c3-439e-9946-3d2e1b364ba0
<img width="1440" alt="Screenshot 2024-05-27 at 9 38 31 PM" src="https://github.com/Expensify/App/assets/102477862/8065c045-4c21-4023-ba3f-2ee74f4454cf">

`to` is `undefined` for some in my high traffic account.

Reply:
fixed